hpat/tests/test_series.py

```diff
@@ -3035,8 +3034,8 @@ def test_impl(series, ascending):
             for index in [gen_srand_array(len(data)), gen_frand_array(len(data)), range(len(data))]:
                 for ascending in [True, False]:
                     series = pd.Series(data, index)
-                    ref_result = test_impl(series, ascending)
-                    jit_result = hpat_func(series, ascending)
+                    ref_result = test_impl(series, ascending, kind='mergesort')
+                    jit_result = hpat_func(series, ascending, kind='quicksort')
```

Comment:
That looks fundamentally wrong because potentially we are able to get different results and then compare these ones. Now it works fine for current input data, but in theory some input can break it.